dist/index.html

```diff
@@ -1,179 +1,175 @@
-						<span class="icon-bar"></span>
-						<span class="icon-bar"></span>
-						<span class="icon-bar"></span>
-						<span class="icon-bar"></span>
-					</button>
-					<a class="brand" href="#"><img src="img/tosdr-icon-16.png" alt="" style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
-					<div class="nav-collapse collapse">
-						<ul class="nav">
-							<li class=""><a href="#services">Ratings</a></li>
-<!--							<li class=""><a href="#classification">Classification</a></li>-->
-<!--							<li class=""><a href="#workinprogress">Progress</a></li>-->
-							<li class=""><a href="https://tosdr.org/about.html">About</a></li>
-<!--							<li class=""><a href="#categories">Categories</a></li>-->
-						</ul>
-<!--						<p class="pull-right">-->
-<!--							<a href="#funding" class="btn btn-success btn-mini" style="position:absolute;">Donate</a>-->
-<!--						</p>-->
-						<p class="pull-right follow-us">
-							Follow us <a href="https://twitter.com/tosdr">@tosdr</a> &nbsp;
-							Donate: <a href="https://opencollective.com/tosdr" target="_blank">On OpenCollective</a>
-						</p>
-					</div>
+<head>
+	<meta charset="utf-8">
+	<title>Terms of Service; Didn&apos;t Read</title>
+	<meta name="author" content="Terms of Service; Didn't Read">
+	<!--Descpription. Should appear on the Search Engines like this.-->
+	<meta name="description" content="Terms of Service; Didn't Read (ToS;DR) is an active project to fix the biggest lie on the web. We help you understand the Terms and Conditions and Privacy Policies of the websites, services and apps you use.">
+	<!--Keywords for Search Engines to rank this page higher.-->
+	<meta name="keywords" content="Terms of Service, Open Source, Free Software, Community, ToSDR, Terms, Privacy, Data, Privacy Policy, Policy, Internet, Project,">
+	<!--OpenGraph Tag, in case it's shared through Social Networks.-->
+	<meta name="og:title" property="og:title" content="Terms of Service; Didn't Read">
+	<!--Summary for Twitter links-->
+	<meta name="twitter:card" content="summary">
+	<!--Canonical Website | Specifies our main url to search engines.-->
+	<link rel="canonical" href="https://tosdr.org/"><!--ONLY IN THE index-template.html file!!!!!!-->
+	<!-- Browser Donations: Flattr -->
+	<meta name="flattr:id" content="qjjrw5">
+	<!-- Browser Donations: LedgerLoops -->
+	<link rel="ledger" href="wss://monetization-hubbie.herokuapp.com">
+	<!-- Browser Donations: Coil -->
+	<meta name="monetization" content="$twitter.xrptipbot.com/michielbdejong" />
+	<!--[if lt IE 9]><script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+	<!--Stylesheets.-->
+	<link href="css/bootstrap.css" rel="stylesheet">
+	<link href="css/bootstrap-responsive.css" rel="stylesheet">
+	<link href="css/custom.css" rel="stylesheet">
+	<link rel="shortcut icon" href="img/tosdr-icon-16.png">
```

Comment:
```suggestion
	<link rel="shortcut icon" href="img/tosdr-icon-57.png">
```
Why not keep the higher res? This tag is favored by many crawlers and a higher resolution is always better